tests/Cache/CacheTest.php

```diff
@@ -63,19 +63,6 @@ public function testDefaults()
         static::assertNull($cache->get($file));
     }
 
-    public function testSetThrowsInvalidArgumentExceptionIfValueIsNotAnInteger()
```

Comment:
:-1:

Reply:
This is important to guard against bad cache values.

Reply:
It's PHPStan's responsibility now (and the class is `@internal`, I didn't make similar changes in public API).